README.md

```diff
@@ -28,4 +28,16 @@ Run ```grunt dev``` to start the app in development mode with client-server live
 
 ### Testing
 
-Run ```grunt test``` to start the karma test runner.
\ No newline at end of file
+TODO: we need to improve testing configuration and related tasks.
```

Comment:
please move this to a new section called known errors in the wiki: https://github.com/MakingSense/mean-seed/wiki and place a link to this wiki on the bottom